YakimS/assignment1/Q1

```diff
@@ -0,0 +1,71 @@
+public class Assignment1Q1
+{
+
+    public static void main(String[] args)
+    {
+        String s1 = "si Apepl Apepl ppppo";
+        String s2 = "ppeAl ppopp Apepl is";
+        System.out.println(isSentenceAnagram(s1, s2,true));
+        System.out.println(s1 + "      " + s2);
+    }
+
+    static boolean isSentenceAnagram(String sentence1, String sentence2, boolean isCaseSensitive)
```

Comment:
Good name of the method that makes it immediately clear what to expect! It would be nice to add comment that provides a bit more details though - what does it mean that two sentences are anagrams?

To make this method usable for code outside of your package it should be made public. Could you make sure that methods intended for external use are public?

Reply:
No problem. All my next assignments will be documented, and I'll make sure to use right access modifier.